### ISSUE TITLE
Stake Ethereum Error reported when staking or unstaking unexpected value

### ISSUE THREAD

**Issue by bosomt** [trezor/trezor-suite#12418]
**Describe the bug**
`Oops! Something went wrong!` reported when staking or unstaking when autofilled or manually filled value out of expected range 

See https://satoshilabs.slack.com/archives/C0543DJBK0C/p1715686025117579

You can see 

```
Error: Error: insufficient funds for gas * price + value: address 0xD048Dcf8d28Ca20F505c0F12664eF66D55998b69 have 210891376600000000 want 271458596977998420
```
in app console.

**Steps to reproduce:**
1. Navigate to stake or unstake if available
2. Try values displayed in screenshots
3. Observe error

**Info:**
 - Suite version: desktop 24.5.2 (1ca8b8063fd7028d1fe242db41caa31c0a1f7d5a)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/24.5.2 Chrome/118.0.5993.159 Electron/27.3.8 Safari/537.36
 - OS: MacIntel
 - Screen: 1512x982
 - Device: Trezor T2T1 2.7.0 regular (revision 1f67c1e55ca2fcf47ce6b28aff75303a8dc19187)
 - Transport: BridgeTransport 2.0.33

**Expected result:**

**Actual result:**
A clear and concise description of what actually happens.

**Screenshots:**
<img width="509" alt="image" src="https://github.com/trezor/trezor-suite/assets/31506317/816a6473-fc5e-41fd-90f9-d639b2aa37ce">

![image](https://github.com/trezor/trezor-suite/assets/31506317/284820ae-49ec-42f7-af4d-75c85fc7845f)

**Comment by tomasklim** [comment]
TODO: 
- Improve error messages

**Comment by MiroslavProchazka** [comment]
Fixed by switching BE to Erigon 2.59.1
We will create separate issue for improving the error message cc @tomasklim

**Comment by bosomt** [comment]
**QA OK**

tested on holesky erigon/2.59.3/linux-amd64/go1.21.6

**Info:**
 - Suite version: desktop 24.6.0 (b6167266c98396f27a6f118acbc9fda04928ebe4)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuiteDev/24.6.0 Chrome/118.0.5993.159 Electron/27.3.8 Safari/537.36
 - OS: MacIntel
 - Screen: 1512x982
 - Device: Trezor T2T1 2.7.0 regular (revision 45e8a842a31e62a6d43d7f6ccac62a45e1198ef0)
 - Transport: BridgeTransport 3.0.0
